heft-plugins/heft-jest-plugin/src/JestPlugin.ts

```diff
@@ -228,9 +230,19 @@ export default class JestPlugin implements IHeftTaskPlugin<IJestPluginOptions> {
       testPathIgnorePatterns: testPathIgnorePatternsParameter.value || pluginOptions?.testPathIgnorePatterns,
       testPathPattern: testPathPatternParameter.value || pluginOptions?.testPathPattern,
       testTimeout: testTimeoutParameter.value ?? pluginOptions?.testTimeout,
-      updateSnapshots: updateSnapshotsParameter.value || pluginOptions?.updateSnapshots
+      updateSnapshots: updateSnapshotsParameter.value || pluginOptions?.updateSnapshots,
+      enableNodeEnvManagement: pluginOptions?.enableNodeEnvManagement || true
```

Comment:
```suggestion
      enableNodeEnvManagement: pluginOptions?.enableNodeEnvManagement ?? true
```
As written the value of `pluginOptions?.enableNodeEnvManagement` would have been completely ignored.